src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,43 @@
 import React from 'react';
+import { Post } from '../../types/Post';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+type Props = {
+  post: Post,
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">
+          {title}
+        </h3>
+
+        {user && (
+          <UserInfo user={user} />
+        )}
+      </div>
+
+      <p className="PostInfo__body">
+        {body}
+      </p>
+
+      <hr />
+
+      { comments.length > 0
+        ? <CommentList comments={comments} />
+        : <b data-cy="NoCommentsMessage">No comments yet</b>}
```

Comment:
```suggestion
      {!!comments.length
        ? <CommentList comments={comments} />
        : <b data-cy="NoCommentsMessage">No comments yet</b>}
```
<img width="817" alt="Screenshot 2023-01-30 at 11 18 59" src="https://user-images.githubusercontent.com/94292654/215437817-56f9edad-fc67-4275-8dea-3464f45ce200.png">